In [9]:
import acquire
import evaluate

### 1 ACQUIRE PHASE

In [ ]:
df = acquire.get_zillow_data()

In [8]:
df.columns

Index(['id', 'bathrooms', 'bedrooms', 'sq_ft', 'taxvaluedollarcnt', 'fips',
       'lotsizesquarefeet', 'propertycountylandusecode', 'regionidcity',
       'regionidzip'],
      dtype='object')

In [5]:
df.head()

id  bathrooms  bedrooms   sq_ft  taxvaluedollarcnt    fips  \
0  1772        2.0       4.0  1604.0           498347.0  6037.0   
1  2028        3.0       3.0  2384.0           549917.0  6037.0   
2  3273        2.0       3.0  1574.0           235272.0  6037.0   
3  3429        2.0       2.0  1619.0           340000.0  6037.0   
4  4505        3.0       2.0  2408.0          2017254.0  6037.0   

   lotsizesquarefeet propertycountylandusecode  regionidcity  regionidzip  
0             6211.0                      0101       12447.0      96415.0  
1             9109.0                      0101       12447.0      96452.0  
2             7538.0                      0100        5534.0      97319.0  
3           111564.0                      0100       40227.0      97329.0  
4            10497.0                      0100       12447.0      96086.0

In [7]:
df.shape

(16035, 10)

# HERE'S WHAT WE GOT:
### df the DataFrame that is ten columns wide and 16035 records long.
### Each row is a house from the table that had a transaction in MAY or JUNE of 2017

# 2 PREP

In [ ]:
## Lets look at Sara's stuff
